#probe_pick_from_board_pose: [0.01581974409099518, -0.20910162982039493, 0.04288628324866296, -0.6956207391833117, 0.7176804072055775, 0.032085655910491645, 0.00413896316646303] # in board frame
probe_action_poses:
  probe_place_pre_holder_pose: [0.6089189648628235, -0.20429673790931702, 0.16731842905282974, 91.61641693115234, 1.9681113958358765, 94.26091766357422]
  probe_place_in_holder_pose: [0.6089189648628235, -0.20429673790931702, 0.11731842905282974, 91.61641693115234, 1.9681113958358765, 94.26091766357422]
  probe_holder_perceive_pose: [0.5871686935424805, -0.2024887502193451, 0.21733148396015167, 176.40609741210938, -0.9146183133125305, 89.14100646972656]
  probe_holder_pre_pick_pose: [0.6270437240600586, -0.1961885243654251, 0.22707584500312805, 177.17393493652344, 1.0941516160964966, 94.44915771484375]
  probe_holder_pick_pose: [0.6348620057106018, -0.19617503881454468, 0.1439492255449295, 177.17367553710938, 1.0925343036651611, 94.44969940185547]
  safe_pose_after_probe_placement: [0.5539224743843079, -0.18827883899211884, 0.13861241936683655, 176.0297088623047, 8.991900444030762, 88.11978149414062]
  tucking_probe_holder_pick: [0.6089082360267639, -0.2042740285396576, 0.0974731370806694, 91.62016296386719, 1.963027000427246, 94.26419067382812]
